src/bin/prepare.sources.sh

```
@@ -0,0 +1,4 @@
+#!/
```

Shellcheck (0.10.0)

[error] This shebang specifies a directory. Ensure the interpreter is a file.

(SC2246)